### ISSUE TITLE
[BUG] ChatGoogleVertexAI response with [object Object]

### ISSUE THREAD

**Issue by timur-nocodia** [FlowiseAI/Flowise#2275]
Flowise 1.6.6

ChatGoogleGenerativeAi stop to work for me - region is not supported (my country is supported by the way) and I decided to use ChatGoogleVertexAi.

I can't get responce from ChatGoogleVertexAi node.
I have no errors in GCC logs.

I used json object (copy - paste from json file) as for my credential.

![Screenshot_1](https://github.com/FlowiseAI/Flowise/assets/120160081/8aedeead-c5e6-4f9b-a46a-79e21a1cda67)

**Comment by toi500** [comment]
You cant "get" the API of Gemini if you qre in an unsupported (country all the EU and UK), but you can use it.

I got my api key on the Gemini AI Studio site via a vpn and I am using it here with no problem.

**Comment by timur-nocodia** [comment]
Yep, thanks. But the thing is that my country is supported and I can get api keys without vpn. But there is no clean answer if my Google Cloud Project location is the country of usage or actually an api call country? It is not clear. My Google Cloud Project is in EU i believe. Anyway, pure api key from ai studio still complaining about my country even it is supported.

**Comment by toi500** [comment]
Ok, I figured out how this works.

I just tested my Gemini API key on a local server, and it does not work. This means that their API key only works if the service is hosted in a supported region, in my case, the USA via Railway. 

Also, it looks like Google lifted the restriction of their API for end users. Weeks ago, even if you had the service running in the USA, the end user could not interact with the API, giving you an error message.
